.lldbinit

```diff
@@ -0,0 +1,8 @@
+# Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+# Exceptions. See /LICENSE for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+command script import external/+llvm_project+llvm-project/llvm/utils/lldbDataFormatters.py
+command script import lldbinit.py
```

Comment:
Can we move this file into the `scripts` directory, just to avoid adding more top-level files? `scripts` is already a bit of a grab-bag of things, so it'll fit there. :)

```suggestion
command script import scripts/lldbinit.py
```

Reply:
Done